data_structures/stacks/Trapping_Rain_Water.py

```diff
@@ -0,0 +1,52 @@
+"""
+	Every element represents the height of the block. The width of the block can be considered 1.
+	Find the volume of trapped rainwater between these block.
+"""
+
+height = [34, 2, 5, 2, 3, 23, 18, 23, 45]
+answer = 162
```

Comment:
These are test inputs, not constants. Move these within the `if __name__ == "__main__":`